### PR TITLE
#163208227 Implement patch vote endpoint

### PR DESCRIPTION
**Pull request by SamueGathua**
**What does this PR do?**
Creates the PATCH  voting endpoint

**Description of the task to be completed**
-Write failing tests for the endpoint
-Create  PATCH voting endpoints
**How can this be manually tested?**
clone this repo https://github.com/SamueGathua/Questioner.git
cd Questioner
Create a virtual environment virtualenv -p python3
Activate the environment source myenv/bin/activate
Run the application flask run.
Test the endpoint on Postman
**What are the relevant PT **
#163208227

upvote and downvote

### CODE REVIEW

**Comment by chepmanu** [comment]
@SamueGathua, remove the `Ft` from your PR title and change the title to something like `Implement patch vote endpoint`, also, a gif image would do a better job to show what you have done. The gif would help others to see what is going on in your task.

**Comment by chepmanu** [comment]
@SamueGathua, you can also add a tag on your PR to indicate the state of where it is. e.g `Finished`, `WIP`, 'Ready for Reviewing`

File: app/api/v1/models/question_models.py [hunk 1]
```diff
@@ -17,13 +17,48 @@ def save(self, meetup_id, question):
...
+            "voted_on" : datetime.datetime.now()
         }
-        voting_record.append(data)
-        return voting_record
+        #Executed when the vote parsed is True
+
+        if vote:
+            data['upvotes'] = data['upvotes'] + 1
+
+        #Executed when the vote parsed is True
```
Comment (chepmanu) [inline app/api/v1/models/question_models.py:44]: This comment seems to imply the opposite, did you mean false instead? Change it to false for it for when a question is downvoted.

Comment (SamueGathua) [reply app/api/v1/models/question_models.py:44]: Thanks alot I have seen it .I will correct and place it  in the line below that executes the donvote

File: app/api/v1/models/question_models.py [hunk 2]
```diff
@@ -17,13 +17,48 @@ def save(self, meetup_id, question):
...
+
+        else:
+            data['downvotes'] = data['downvotes'] + 1
+        self.rec.append(data)
```
Comment (SamueGathua) [inline app/api/v1/models/question_models.py:48]: I have seen that I am implementing the downvote wrongly.Here I have +1 instead of -1

**Pull request merged**